### PR TITLE
clouds/aws: add methods for retrieving ec2 types info

### PR DESCRIPTION
**Pull request by saheienko**

### CODE REVIEW

File: pkg/clouds/aws/instances.go [hunk 1]
```diff
@@ -46,23 +52,48 @@ func New(keyID, secret string) (*Client, error) {
...
+		}
+
+		ec2Infos = append(ec2Infos, info)
+	}
+
+	return ec2Infos, nil
+}
+
+// CreateInstance starts a new instance due to the config.
+func (c *Client) CreateInstance(ctx context.Context, cfg InstanceConfig) (*ec2.Instance, error) {
```
Comment (egopher) [inline pkg/clouds/aws/instances.go]: Better make tags part of InstanceConfig

Comment (saheienko) [reply pkg/clouds/aws/instances.go]: Tags is a part of InstanceConfig.

File: build/gofmt.sh [hunk 2]
```diff
@@ -13,9 +13,10 @@ find_files() {
 }
 
 GOFMT="gofmt -s"
-bad_files=$(find_files | xargs $GOFMT -l)
```
Comment (stgleb) [inline build/gofmt.sh]: Is it unrelated changes?

Comment (saheienko) [reply build/gofmt.sh]: Yeap, `make fmt` was broken by https://github.com/supergiant/supergiant/pull/587.

Comment (stgleb) [reply build/gofmt.sh]: Good, I also have noticed that. Maybe you can create separate PR for that so we can merge it ASAP. But it is not critical for me..
